ansible/roles/image_builder/tasks/main.yaml

```
@@ -0,0 +1,41 @@
+  ansible.builtin.copy:
+    src: /tmp/built-image/eksa-os.qcow2
+    dest: /var/www/html/eksa-os.qcow2
+    mode: '0644'
+  become: true
+
+- name: Ensure web server is running
+  ansible.builtin.service:
+    name: apache2
+    state: started
+    enabled: true
+  become: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)